README.md

```diff
@@ -60,7 +62,7 @@ or the managed Azure API for FHIR:
 .\Create-FhirServerSamplesEnvironment.ps1 -EnvironmentName <ENVIRONMENTNAME> -UsePaaS $true
 ```
 
-and to include the ADF pipeline:
+and to include the ADF pipeline if using the PaaS Impl.:
```

Comment:
ADF does not depend on using the PaaS version of the FHIR service. If you have problems deploying with OSS FHIR server and ADF, please put an issue with a repro so we can investigate.